packages/uniorg-parse/src/parser.ts

```diff
@@ -1486,6 +1491,16 @@ class Parser {
     return u('strike-through', { contentsBegin, contentsEnd }, []);
   }
 
+  private parseStatisticsCookie(): StatisticsCookie | null {
+    const m = this.r.lookingAt(/\[[0-9]*(\%|\/[0-9]*)\]/);
```

Comment:
nit: you can use `this.r.advance(this.r.lookingAt(...))` to avoid resetting offset manually.